ballerina/utils.bal

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+import ballerina/lang.'string as strings;
+
+isolated function getSubstring(string str, int startIndex) returns string {
+    if startIndex == 0 {
+        return str;
+    }
+    return strings:substring(str, startIndex, str.length());
+}
```

Comment:
Do we need this function? Can't we just use `strings:substring`

Reply:
We can call this, but this is used in `replace` and `search` functions. Hence, this function is used to avoid duplicate code.

Reply:
Do we need to check `startIndex == 0` this. it is already handled in `strings:substring` right?

Reply:
Yes, `strings:substring` handles the error scenario, but if the start index is 0, we don't need to get a substring.